packages/website/components/navbar.js

```diff
@@ -39,7 +39,7 @@ export default function Navbar({ bgColor = '', isLoggedIn, isLoadingUser }) {
             onClick={onLinkClick}
           >
             <img src="/w3storage-logo.svg" style={{ height: '1.8rem' }} />
-            <span className="space-grotesk ml-2 text-w3storage-purple font-medium text-md hidden xl:inline-block">Web3.Storage</span>
+            <span className="space-grotesk ml-2 text-w3storage-purple font-medium typography-logo hidden xl:inline-block">Web3.Storage</span>
```

Comment:
You can just use `text-base` instead of adding a new typography class

Reply:
You're right, didn't see the text-base, my bad :)